javascript/safari-driver/inject/commands/commands.js

```diff
@@ -503,6 +517,7 @@ safaridriver.inject.CommandRegistry.getInstance()
         CommandName.SET_WINDOW_SIZE, commands.setWindowSize,
         CommandName.SUBMIT_ELEMENT, commands.submitElement,
         CommandName.SWITCH_TO_FRAME, commands.switchToFrame,
+        CommandName.GET_CANVAS_URL,commands.getCanvasUrl,
```

Comment:
Since this command depends on the html2canvas library, it should be defined in a separate module that gets loaded on command.